qutebrowser/mainwindow/statusbar/url.py

```diff
@@ -153,7 +153,7 @@ def set_hover_url(self, link, _title, _text):
             _text: The text of the hovered link (string)
         """
         if link:
-            self._hover_url = link
+            self._hover_url = QUrl(link).toString()
```

Comment:
I think it'd be better to use `toDisplayString()` so e.g. passwords aren't shown.